samples/WindowsAlarmsAndClock/Pages/AlarmPage.cs

```diff
@@ -0,0 +1,107 @@
+namespace WindowsAlarmsAndClock.Pages
+{
+    using System;
+    using System.Collections.ObjectModel;
+    using System.Linq;
+
+    using Legerity.Pages;
+    using Legerity.Windows.Extensions;
+
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    using OpenQA.Selenium;
+    using OpenQA.Selenium.Appium;
+
+    /// <summary>
+    /// Defines the alarm page of the Windows Alarms & Clock application.
```

Comment:
Can you fix this invalid XML? Believe this should be `&amp;`

Reply:
This should be fixed now